tests/dataaccess/queryorder/queryorder.csv

```diff
@@ -0,0 +1,24 @@
+action,ms,bytes,lang,code,repeat,minver,comment
```

Comment:
Is this all the tests we have for this?  Does it really cover all scenarios?

Reply:
The piece of coded we are testing, [Queryorder](https://github.com/AquaQAnalytics/TorQ/blob/dataaccessapi/code/dataaccess/queryorder.q), is a really light piece of code- it only reorders the functional select.